2016-07-31 03:09:09,306 - streamparse.storm.component - ERROR - Exception in WordCounter.run()
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/streamparse/storm/component.py", line 489, in run
    self._run()
  File "/usr/lib/python2.7/site-packages/streamparse/storm/bolt.py", line 219, in _run
    self.process(tup)
  File "bolts/wordcount.py", line 36, in process
    cur.execute("UPDATE Tweetwordcount SET count=%s WHERE word=%s", (self.counts[word], word));
InternalError: current transaction is aborted, commands ignored until end of transaction block

